Dockerfile

```diff
@@ -28,9 +28,13 @@ RUN set -ex; \
         g++ \
         gnupg \
     '; \
+    runtimeDeps=' \
```

Comment:
i moved the installation of libjemalloc2 as well so that it is not being re-installed everytime there's a code change in ./snuba/